src/main/java/mate/academy/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Size;
+
+public record UserLoginRequestDto(
+        @NotEmpty
+        @Size(min = 8, max = 50)
```

Comment:
In UserRegistrationRequest class, you set 4
```suggestion
        @Size(min = 4, max = 50)
```